tests/testthat/_snaps/options.md

```diff
@@ -23,5 +24,5 @@
     - strictly_immutable: input must be TRUE or FALSE.
     - debug_polars: input must be TRUE or FALSE.
     
-    More info at `?polars_options`.
+    More info at `?polars::polars_options`.
```

Comment:
Note: This is not directly related to this PR, but it is necessary to be able to display the help page without loading the package because the package cannot be loaded if the option validation fails.